### DISCUSSION TITLE
Quad Bike Object Detection web app with YOLOv8 and Flask!

### DISCUSSION THREAD

**Discussion by tanwiraabideen** [ultralytics/ultralytics#14221] [Show and tell]
Here is a link to the GitHub repo for the web application I made using Flask and Yolov8 to help ATV owners maintain their vehicles themselves.

![Screenshot 2024-07-05 at 9 32 03 AM](https://github.com/ultralytics/ultralytics/assets/155897447/19cec1cd-7377-430d-babc-d912af08c409)

[GitHub Repo](https://github.com/tanwiraabideen/ATV-OD-YOLOv8)

**Comment by glenn-jocher** [comment]
Hi @tanwiraabideen,

Thank you for sharing your project! It's fantastic to see YOLOv8 being used in such a practical and innovative way to assist ATV owners. Your web application looks impressive, and I'm sure it will be incredibly useful for many users.

If you have any questions or run into any issues while developing or deploying your application, feel free to ask. We're here to help!

Best of luck with your project! 🚀
